README.md

```diff
@@ -1,282 +1,584 @@
 
+  
+
 ### JIRA_ASSIGNEE
 
+  
+
 **Required: No**
 
+  
+
 **Default Value: N/A**
 
+  
+
 Assignee to be used for any ticket created by this tool.
 
+  
+
 ### JIRA_CUSTOM_FIELDS
 
+  
+
 **Required: No**
 
+  
+
 **Default Value: N/A**
 
+  
+
 JSON string of Jira custom field keys/values, e.g. `{ "customfield_14117": "example-value" }`
 
+  
+
 ### AWS_SEVERITIES
 
+  
+
 **Required: No**
 
+  
+
 **Default Value: 'CRITICAL,HIGH,MEDIUM'**
 
+  
+
 Comma separated list of AWS Security Hub finding severities for which this tool should create Jira Issues.
 
+  
+
 ### AWS_REGION
 
+  
+
 **Required: No**
 
+  
+
 **Default Value: us-east-1**
 
+  
+
 AWS Region to target
 
+  
+
 ### SECURITY_HUB_NEW_ISSUE_DELAY
 
+  
+
 **Required: No**
 
+  
+
 **Default Value: 86400000**
 
+  
+
 Delay in milliseconds for filtering out ephemeral issues. The default value of 86400000 is 24 hours.
 
+  
+
 ### AUTO_CLOSE
 
+  
+
 **Required: No**
 
+  
+
 **Default Value: true**
 
+  
+
 If set to true, when a Security Hub finding is resolved, the corresponding Jira issue will be closed. If set to false, the description will be updated and a comment will be added indicating the finding has been resolved, but the Jira Issue status will not change. The closure has following modes
 
+  
+
 **Full Transition Handling**
+
 The **Full Transition Handling** feature operates automatically when the `jira-transition-map` is not specified. It helps manage issue resolution by automatically transitioning issues through the workflow towards a final status.
 
+  
+
 ### How It Works
 
-- **Automatic Transitioning**: The feature attempts to move the issue to one of the commonly used closing statuses, which are:
+  
+
+-  **Automatic Transitioning**: The feature attempts to move the issue to one of the commonly used closing statuses, which are:
+
+  
+
+-  `done`
+
+-  `closed`
+
+-  `close`
+
+-  `complete`
+
+-  `completed`
 
-  - `done`
-  - `closed`
-  - `close`
-  - `complete`
-  - `completed`
-  - `deploy`
-  - `deployed`
+-  `deploy`
 
-- **Stopping Point**: If the workflow does not include any of these closing statuses, the system will transition the issue to the final status in the workflow that is not a reject status.
-- **Reject Statuses**: The system avoids transitioning to statuses that indicate rejection or cancellation, such as:
+-  `deployed`
 
-  - `canceled`
-  - `backout`
-  - `rejected`
-  - `cancel`
-  - `reject`
-  - `block`
-  - `blocked`
+  
+
+-  **Stopping Point**: If the workflow does not include any of these closing statuses, the system will transition the issue to the final status in the workflow that is not a reject status.
+
+-  **Reject Statuses**: The system avoids transitioning to statuses that indicate rejection or cancellation, such as:
+
+  
+
+-  `canceled`
+
+-  `backout`
+
+-  `rejected`
+
+-  `cancel`
+
+-  `reject`
+
+-  `block`
+
+-  `blocked`
+
+  
 
 For example If there's a workflow which has following transition rules
 
+  
+
```

Comment:
This ASCII diagram seems a bit confusing. Any chance you can mock this up on something like diagrams.net and replace it with a screenshot?

Reply:
Yes, replaced with screenshot